common/men_at_arms_types/00_maa_types.txt

```diff
@@ -49,6 +49,7 @@ bowmen = {
 
```

Comment:
I don't get. Do you follow [this design](https://docs.google.com/spreadsheets/d/1vEueLb4GC9az8ayDmweFdA5LPHifqCQRMEihr8-tY_Q/edit?usp=sharing) or not. If you do, `light_footmen` should counter `caster_unit`.